hmcli/base/commands.py

```diff
@@ -0,0 +1,15 @@
+from importlib import import_module
+from pkgutil import iter_modules
+
+import click
+
+import hmcli
+
+@click.group()
+def hmcli_cmds():
+    pass
+
+def main():
+  for plugin in iter_modules(hmcli.__path__, hmcli.__name__ + "."):
+    import_module(plugin.name)
```

Comment:
nitpick, in hivemind context plugins will refer to ovos plugins 99% of the time, so let's avoid that wording